src/client/java/teammates/client/scripts/sql/VerifyNonCourseEntityCounts.java

```diff
@@ -0,0 +1,99 @@
+package teammates.client.scripts.sql;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.cloud.datastore.QueryResults;
+import com.googlecode.objectify.cmd.Query;
+
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Root;
+
+import teammates.client.connector.DatastoreClient;
+import teammates.client.util.ClientProperties;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.entity.Account;
+import teammates.storage.entity.BaseEntity;
+
+public class VerifyNonCourseEntityCounts extends DatastoreClient {
+    private VerifyNonCourseEntityCounts() {
+        String connectionUrl = ClientProperties.SCRIPT_API_URL;
+        String username = ClientProperties.SCRIPT_API_NAME;
+        String password = ClientProperties.SCRIPT_API_PASSWORD;
+
+        HibernateUtil.buildSessionFactory(connectionUrl, username, password);
+    }
+
+    public static void main(String[] args) throws Exception {
+        new VerifyNonCourseEntityCounts().doOperationRemotely();
+    }
+
+    private void printEntityVerification(String className, int datastoreCount, long psqlCount) {
+        System.out.println("========================================");
+        System.out.println(className);
+        System.out.println("Objectify count: " + datastoreCount);
+        System.out.println("Postgres count: " + psqlCount);
+        System.out.println("Correct number of rows?: " + (datastoreCount == psqlCount));
+    }
+
+
+    private Long countPostgresEntities(Class<? extends teammates.storage.sqlentity.BaseEntity> entity) {
+        HibernateUtil.beginTransaction();
```

Comment:
Might not need a transaction, but small issue

Reply:
<img width="1145" alt="image" src="https://github.com/TEAMMATES/teammates/assets/25302138/a80b6ef0-4774-4872-a368-f462dcee774e">

Got this error. I can reduce this to a single transaction in `doOperation()` if you prefer, but I was just thinking keep them separate